scripts/docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
   mc:
     depends_on:
       - minio
-    image: minio/mc
+    image: minio/mc:RELEASE.2025-04-16T18-13-26Z-cpuv1
```

Comment:
Thanks, I think we just ran into this as well with @carlopi , he needed to change the `mc config` line to use `mc alias` instead, as the `mc config` command was deprecated+removed on the version of `minio/mc` that he pulled